### PR TITLE
added papers recommanded by Son about how to represent actions

### PR DESCRIPTION
**Pull request by EtienneTignon**

### CODE REVIEW

**Review by tortinator — CHANGES_REQUESTED** [review]
Merci @EtienneTignon . Please format the entries along our conventions (see `README.md`)

**Comment by tortinator** [comment]
Salut @EtienneTignon : What about your pull request? @tortinator

**Comment by EtienneTignon** [comment]
Hi @tortinator . Sorry for the delay. I'm still on it.

**Comment by tortinator** [comment]
No worries, @EtienneTignon, I just wanted to make sure that it is not forgotten :)

File: krr.bib [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@comment{-_-_-_-_-_-_-_-_-_-_-_-_-feb_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_}
```
Comment (tortinator) [inline krr.bib:1]: What's this?

File: krr.bib [hunk 2]
```diff
@@ -1910,6 +1912,14 @@ @article{basotainsc13a
   year = {2013}
 }
 
+@inproceedings{basotu2002a,
```
Comment (tortinator) [inline krr.bib:1915]: 2002 -> 02

File: krr.bib [hunk 3]
```diff
@@ -1910,6 +1912,14 @@ @article{basotainsc13a
...
+  title = {A Transition Function Based Characterization of Actions with Delayed and Continuous Effects.},
+  author = {C. Baral and T, Son and L. Tuan},
+  month = {01},
```
Comment (tortinator) [inline krr.bib:1918]: delete

File: krr.bib [hunk 4]
```diff
@@ -1910,6 +1912,14 @@ @article{basotainsc13a
...
+  pages = {291-302},
+  year = {2002}
+}
```
Comment (tortinator) [inline krr.bib:1921]: Where has it appeared?

File: krr.bib [hunk 5]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
   year = {2002}
 }
 
+@article{cafogi2012a,
+  title = {Planning as quantified boolean formula},
+  author = {M. Cashmore and M. Fox and E. Giunchiglia},
+  doi = {10.3233/978-1-61499-098-7-217},
```
Comment (tortinator) [inline krr.bib:4793]: delete

File: krr.bib [hunk 6]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
...
+  journal = {Frontiers in Artificial Intelligence and Applications},
+  month = {01},
```
Comment (tortinator) [inline krr.bib:4795]: delete

File: krr.bib [hunk 7]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
   year = {2002}
 }
 
+@article{cafogi2012a,
```
Comment (tortinator) [inline krr.bib:4790]: Is this an article in a journal..?

File: krr.bib [hunk 8]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
...
+  author = {M. Cashmore and M. Fox and E. Giunchiglia},
+  doi = {10.3233/978-1-61499-098-7-217},
+  journal = {Frontiers in Artificial Intelligence and Applications},
+  month = {01},
+  pages = {217-222},
+  volume = {242},
+  year = {2012}
+}
+
+@inbook{cafoloda2016a,
```
Comment (tortinator) [inline krr.bib:4801]: 2016  -> 16

File: krr.bib [hunk 9]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
...
+  title = {A Compilation of the Full {PDDL}+ Language into {SMT}},
+  author = {M. Cashmore and M. Fox and D. Long and D. Magazzeni},
+  abstract = {Planning in hybrid systems is important for dealing with realworldapplications. PDDL+ supports this representation ofdomains with mixed discrete and continuous dynamics, andsupports events and processes modeling exogenous change.Motivated by numerous SAT-based planning approaches, wepropose an approach to PDDL+ planning through SMT, describingan SMT encoding that captures all the features of thePDDL+ problem as published by Fox and Long (2006). Theencoding can be applied on domains with nonlinear continuouschange. We apply this encoding in a simple planningalgorithm, demonstrating excellent results on a set of benchmarkproblems.},
```
Comment (tortinator) [inline krr.bib:4804]: delete

File: krr.bib [hunk 10]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
...
+  booktitle = {Twenty-Sixth International Conference on Automated Planning and Scheduling},
+  day = {12},
```
Comment (tortinator) [inline krr.bib:4806]: delete

File: krr.bib [hunk 11]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
...
+  language = {English},
```
Comment (tortinator) [inline krr.bib:4807]: delete

File: krr.bib [hunk 12]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
...
+  month = jun,
```
Comment (tortinator) [inline krr.bib:4808]: delete

File: krr.bib [hunk 13]
```diff
@@ -4777,6 +4787,29 @@ @article{cafito02a
...
+  pages = {217-222},
+  volume = {242},
+  year = {2012}
+}
+
+@inbook{cafoloda2016a,
+  title = {A Compilation of the Full {PDDL}+ Language into {SMT}},
+  author = {M. Cashmore and M. Fox and D. Long and D. Magazzeni},
+  abstract = {Planning in hybrid systems is important for dealing with realworldapplications. PDDL+ supports this representation ofdomains with mixed discrete and continuous dynamics, andsupports events and processes modeling exogenous change.Motivated by numerous SAT-based planning approaches, wepropose an approach to PDDL+ planning through SMT, describingan SMT encoding that captures all the features of thePDDL+ problem as published by Fox and Long (2006). Theencoding can be applied on domains with nonlinear continuouschange. We apply this encoding in a simple planningalgorithm, demonstrating excellent results on a set of benchmarkproblems.},
+  booktitle = {Twenty-Sixth International Conference on Automated Planning and Scheduling},
```
Comment (tortinator) [inline krr.bib:4805]: crossref via procs.bib?

**Review by tortinator — CHANGES_REQUESTED** [review]

File: krr.bib [hunk 14]
```diff
@@ -8201,6 +8210,36 @@ @article{foxlon03a
...
+  archiveprefix = {arXiv},
+  bibsource = {dblp computer science bibliography, https://dblp.org},
+  biburl = {https://dblp.org/rec/journals/corr/abs-1110-2200.bib},
+  eprint = {1110.2200},
+  journal = {Computing Research Repository},
+  timestamp = {Mon, 13 Aug 2018 16:46:52 +0200},
+  url = {http://arxiv.org/abs/1110.2200},
+  volume = {abs/1110.2200},
+  year = {2011}
+}
```
Comment (tortinator) [inline krr.bib:8225]: compare with existing arxiv entries - this is no proper article, just a miscellaneous entry

File: krr.bib [hunk 15]
```diff
@@ -8201,6 +8210,36 @@ @article{foxlon03a
...
+  interhash = {3aa074923e627c76bfa367db90f5aa75},
+  intrahash = {e14097ce73d3b1f4ee59a7389c9579e9},
+  journal = {Computing Research Repository},
+  keywords = {pddl},
+  timestamp = {2012-01-17T17:32:22.000+0100},
+  url = {http://dblp.uni-trier.de/db/journals/corr/corr1106.html#abs-1106-4561},
+  volume = {abs/1106.4561},
+  year = {2011}
+}
+
```
Comment (tortinator) [inline krr.bib:8242]: same as above

**Review by tortinator — CHANGES_REQUESTED** [review]

**Comment by tortinator** [comment]
Your branch is now about 80 commits behind. You should merge in the current master.

**Review by tortinator — CHANGES_REQUESTED** [review]
This is inacceptable - you really need to get your act together!

**Pull request closed without merging**